### PR TITLE
Emily's Week1 homework exercises

### PR DESCRIPTION
**Pull request by metroretro**
Updated rspec file with passing tests and answered homework questions on objects, classes and strings.

### CODE REVIEW

**Comment by metroretro** [comment]
Added week 2 homework

File: week1/homework/questions.txt [hunk 1]
```diff
@@ -1,15 +0,0 @@
-Please read:
```
Comment (reneedv) [inline week1/homework/questions.txt:1]: Do you have these answered somewhere else? I don't see your answers here.

**Comment by reneedv** [comment]
Just need week 1's questions. Otherwise, good work!

**Comment by metroretro** [comment]
Arg! I do have Week 1 questions answered. I messed up something with my git
pull/push. I removed the file from my local clone so when I pushed to
github I must've lost the previous version. I'll figure out how to restore
a previous version and get that over to you :)

Thanks,

Emily

On Thu, Oct 16, 2014 at 4:32 PM, Renée Hendricksen <notifications@github.com

**Pull request closed without merging**